src/main/_config.yml

```diff
@@ -52,66 +52,58 @@ asciidoc_attributes: &asciidoc_attributes
+  identity-provider: Keycloak
   identity-provider-url: https://www.keycloak.org/
   identity-provider-version: 6.0
-  identity-provider: Keycloak
-  identity-provider2: Keycloak
-  image-puller-tag: latest
   image-puller: quay.io/eclipse/kubernetes-image-puller
-  theia-endpoint-image: eclipse/che-theia-endpoint-runtime:next
-
-# URLs
-  link-installing-an-instance: link:{site-baseurl}che-7/che-quick-starts/[Che 'quick-starts']
+  image-puller-tag: latest
   link-accessing-a-git-repository-via-https: link:{site-baseurl}che-7/version-control/#accessing-a-git-repository-via-https_version-control[Accessing a Git repository via HTTPS]
   link-adding-tls-certificates: link:{site-baseurl}che-7/installing-che-in-tls-mode-with-self-signed-certificates[Installing {prod-short} in TLS mode with self-signed certificates]
   link-advanced-configuration-options-for-the-che-server: link:{site-baseurl}che-7/advanced-configuration-options-for-the-che-server-component/#understanding-che-server-advanced-configuration-using-the-operator_advanced-configuration-options-for-the-che-server-component[Advanced configuration options for the Che server component]
   link-advanced-configuration-options: link:{site-baseurl}che-7/advanced-configuration-options-for-the-che-server-component/[Advanced configuration options for the Che server component]
   link-building-a-custom-plug-in-registry: link:{site-baseurl}che-7/building-and-running-a-custom-registry-image/#building-a-custom-devfile-registry_building-and-running-a-custom-registry-image[Building a custom plug-in registry]
   link-cli-github: https://github.com/che-incubator/chectl
+  link-installing-an-instance: link:{site-baseurl}che-7/che-quick-starts/[Che 'quick-starts']
   link-installing-on-openShift-3-using-the-operator: link:{site-baseurl}che-7/installing-{prod-id-short}-on-openshift-3-using-the-operator[Installing {prod-short} on OpenShift 3 using the Operator]
   link-limits-for-user-workspaces: link:{site-baseurl}che-7/advanced-configuration-options-for-the-che-server-component/#users-workspace-limits[Limits for user workspaces]
   link-postgres-dockerfile-location: https://github.com/eclipse/che/tree/master/dockerfiles/postgres
   link-server-identity-provider-dockerfile-location: https://github.com/eclipse/che/tree/master/dockerfiles/keycloak
-  url-devfile-registry-repo: https://github.com/eclipse/che-devfile-registry
-  url-plug-in-registry-repo: https://github.com/eclipse/che-plugin-registry
-  # intentionally blank:
-  repo-path: ''
-
-# stack names
   nodejs-stack: nodejs
-
   oc4-ver: 4.4
-  ocp: OpenShift{nbsp}Container{nbsp}Platform
   ocp3-ver: 3.11
   ocp4-ver: 4.5
+  ocp: OpenShift{nbsp}Container{nbsp}Platform
   orch-cli: kubectl
   orch-name: Kubernetes
   orch-namespace: namespace
   osd: OpenShift{nbsp}Dedicated
+  postgresql-image-url: quay.io/eclipse/che-postgres
+  prod2: Eclipse{nbsp}Che
   prod-checluster: eclipse-che
   prod-cli: chectl
   prod-deployment: che
+  prod: Eclipse{nbsp}Che
```

Comment:
looks like this one is not sorted

Reply:
That's the result of sorting with the `sort` command ...

Reply:
I don't know which `sort` command


```
$ cat /tmp/list| sort -d                                                                                                                                                                    
  oc4-ver: 4.4
  ocp: OpenShift{nbsp}Container{nbsp}Platform
  ocp3-ver: 3.11	  ocp3-ver: 3.11
  ocp4-ver: 4.5	  ocp4-ver: 4.5
  orch-cli: kubectl	  orch-cli: kubectl
  orch-name: Kubernetes	  orch-name: Kubernetes
  orch-namespace: namespace	  orch-namespace: namespace
  osd: OpenShift{nbsp}Dedicated	  osd: OpenShift{nbsp}Dedicated
  postgresql-image-url: quay.io/eclipse/che-postgres
  prod: Eclipse{nbsp}Che
  prod2: Eclipse{nbsp}Che
  prod-checluster: eclipse-che	  prod-checluster: eclipse-che
  prod-cli: chectl	  prod-cli: chectl
  prod-deployment: che	  prod-deployment: che
  prod-host: che-host	  prod-host: che-host
```

```
$ cat /tmp/list| sort -d                                                                                                                                                                 
  oc4-ver: 4.4
  ocp: OpenShift{nbsp}Container{nbsp}Platform
  ocp3-ver: 3.11	  ocp3-ver: 3.11
  ocp4-ver: 4.5	  ocp4-ver: 4.5
  orch-cli: kubectl	  orch-cli: kubectl
  orch-name: Kubernetes	  orch-name: Kubernetes
  orch-namespace: namespace	  orch-namespace: namespace
  osd: OpenShift{nbsp}Dedicated	  osd: OpenShift{nbsp}Dedicated
  postgresql-image-url: quay.io/eclipse/che-postgres
  prod: Eclipse{nbsp}Che
  prod2: Eclipse{nbsp}Che
  prod-checluster: eclipse-che
  prod-cli: chectl	  prod-cli: chectl
  prod-deployment: che	  prod-deployment: che
  prod-host: che-host	  prod-host: che-host
```